packages/components/src/components/popup/popup-footer.tsx

```diff
@@ -1,8 +1,12 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const PopupFooter = ({ children, has_separator }) => (
+type TPopupFooter = {
+    children: React.ReactNode;
+    has_separator: boolean;
+};
+
+const PopupFooter = ({ children, has_separator }: TPopupFooter) => (
```

Comment:
```suggestion
const PopupFooter = ({ has_separator }: React.PropsWithChildren<TPopupFooter>) => (
```

Reply:
It seems like if I replace props with `React.PropsWithChildren` it will return error stating that `children` doesn't exist 🙏

Reply:
if you are using `React.PropsWithChildren`, you need to keep the children inside PopupFooter props but don't need to have it inside your type attributes. I think this is the reason that you get error

Reply:
owh yea this solved the issue thanks @shayan-deriv 🙏

Reply:
@shayan-deriv yes he is right my bad

Reply:
still I'm seeing that we don't use React.propswithchildren

Reply:
updated the component 🙏